pkg/backingstore/reconciler.go

```diff
@@ -154,6 +158,9 @@ func (r *Reconciler) Reconcile() (reconcile.Result, error) {
 		return reconcile.Result{}, nil // final state
 	}
 
+	//Should we fail/ignore in case the config map is not present?
+	util.KubeCheck(r.CoreAppConfig)
```

Comment:
shouldn't this happen only for pvpool? and then maybe you can throw error if it doesn't exist